webportal/webapp/src/environments/environment.ts

```diff
@@ -4,13 +4,14 @@
 // The list of which env maps to which file can be found in `.angular-cli.json`.
 
 export const environment = {
-  production: '',
```

Comment:
Lets not check this in... I guess...

Reply:
I added the blank LOGIN_GOV_COGNITO_APP_CLIENT_ID var, and then the rest is linting.